pkg/sql/parsers/dialect/mysql/mysql_sql.y

```diff
@@ -7189,6 +7191,12 @@ index_def:
         if $3[1] != "" {
                t := strings.ToLower($3[1])
             switch t {
+ 	    case "btree":
```

Comment:
what is this?   just fake mysql compatibility?

Reply:
Yes. These enums were already defined in the [create.go](https://github.com/matrixorigin/matrixone/blame/53178af5a27b9fa73748ada2f92f38b771a8ed83/pkg/sql/parsers/tree/create.go#L747). I have linkedin them in the parser. In the future, I will be defining "IVF" for vector index.